src/main/java/th/ac/kmitl/it/se/soa2018/group1/TaxInvoiceWebApplication/model/PostalTradeAddress.java

```diff
@@ -0,0 +1,58 @@
+package th.ac.kmitl.it.se.soa2018.group1.TaxInvoiceWebApplication.model;
```

Comment:
ทำไมชื่อมีทั้ง snake-case และ camel-case หล่ะ

Reply:
หมายถึงตรงไหนหรอครับ

Reply:
"th.ac.kmitl.it.se.soa2018.group1" => snake-case
TaxInvoiceWebApplication => camel-case